app/src/main/res/layout/collaboration_recyclerview_item.xml

```diff
@@ -0,0 +1,35 @@
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
bro in every layout related XML files there should be `<?xml version="1.0" encoding="utf-8"?>`